spec/pagerduty_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Pagerduty do
+  describe ".build" do
+    When(:build) { Pagerduty.build(config) }
+
+    context "given an integration key and API version: `1`" do
+      Given(:config) { { integration_key: "test-key", api_version: 1 } }
```

Comment:
It's been ages since I see rspec-given DSL

Reply:
Yeah, it's not very popular. I do like some of its features though.